_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: The psychology of perception tests an age-old fashion foible
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](https://en.wikipedia.org/wiki/The_dress) sparked the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
+The answer may seem obvious, but experimental results and a well-documented illusion contradict most people's intuition that horizontal stripes make the wearer look wider.
+
+<!--break-->
+
+As pioneering vision scientist Helmholtz [noted back in 1867](http://poseidon.sunyopt.edu/helmholtz/OCRVolume3.pdf) (see page 206!), a person wearing horizontal stripes should, contrary to popular belief, appear _thinner_ than one with vertical stripes.
+This observation is the basis for the [Helmholtz illusion](http://www.bbc.co.uk/radio4/features/sywtbas/finalists/stripes/), at least as it applies to shapes, not bodies.
+A team of researchers from Japan tested whether the size of the body wearing stripes changes the strength of the effect.
+
+[In line with previous research](http://ipe.sagepub.com/content/2/1/69), they found that the common idea that horizontal stripes make one look fat is in fact wrong.
+In the experiment, a group of students rated computer-generated figures wearing either a horizontally or vertically striped shirt.
+The figures had different degrees of thin and fat body shapes to determine when the horizontal and vertical-striped figures looked ‘equally fat’ for each observer.
+This measurement is a baseline for each person’s perception, and all participants perceived a person with horizontal stripes to be thinner than a person with vertical stripes.
+This agrees with the Helmholtz illusion phenomenon, that a horizontally striped square is perceived as ‘thinner’ than one with vertical stripes.
```

Comment:
How about "with the phenomenon of the Helmholtz illusion" rather than the unhyphenated compound modifier? Your call, no strong feelings on my side.